ui/components/app/signature-request/signature-request.component.js

```diff
@@ -17,7 +17,6 @@ export default class SignatureRequest extends PureComponent {
       name: PropTypes.string,
     }).isRequired,
     isLedgerWallet: PropTypes.bool,
-    clearConfirmTransaction: PropTypes.func.isRequired,
```

Comment:
This prop should be removed in `signature-request.container.js` as well, since it's no longer needed. The entire `mapDispatchToProps` function can be omitted, since this is the only function it returns.